docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -1,51 +1,81 @@
 [[getting-started]]
-== Quick start: Add data using Elasticsearch APIs
+== Quick start: Indices, documents, and mappings
 ++++
-<titleabbrev>Basics: Add data using APIs</titleabbrev>
+<titleabbrev>Basics: Indices, documents, and mappings</titleabbrev>
 ++++
 
 In this quick start guide, you'll learn how to do the following tasks:
```

Comment:
One element I think we should ensure we talk about is pre-reqs.. in case this person hit this article they may not realize they need to have an environment to use. We don't need to put the install instructions in here but at least a reference on where to go to setup their environment.

Reply:
Added in https://github.com/elastic/elasticsearch/pull/113558/commits/31d8006392b2eb3e578309b43493ad420c6a99e6